dspace-server-webapp/src/main/java/org/dspace/app/rest/model/wrapper/TemplateItem.java

```diff
@@ -0,0 +1,48 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.app.rest.model.wrapper;
+
+import java.util.Date;
+import java.util.List;
+import java.util.UUID;
+
+import org.dspace.content.Collection;
+import org.dspace.content.Item;
+import org.dspace.content.MetadataValue;
+
+public class TemplateItem {
```

Comment:
This Class is missing JavaDocs. It should describe what a "TemplateItem" is by saying something like "Represents a Collection's Item Template"